homeassistant/components/matter/number.py

```diff
@@ -33,6 +33,30 @@
 from .helpers import get_matter
 from .models import MatterDiscoverySchema
 
+# intensities in inovelli LEDs are represented by a number from 0 to 75 but only 11
+# values are actually used when one of 11 levels is set by manually configuring the dimmer
+INOVELLI_LED_INTENSITY_MAP = {
+    0: 0,
+    1: 1,
+    2: 8,
+    3: 15,
+    4: 24,
+    5: 33,
+    6: 42,
+    7: 50,
+    8: 59,
+    9: 68,
+    10: 75,
+}
+
+INOVELLI_LED_INTENSITY_MAP_REVERSE = {}
```

Comment:
We can simply iterate `INOVELLI_LED_INTENSITY_MAP` to create the reverse map. Search for reverse in the project source to find examples.

Reply:
```>>> print(INOVELLI_LED_INTENSITY_MAP_REVERSE)
{0: 0, 1: 1, 2: 1, 3: 1, 4: 1, 5: 1, 6: 1, 7: 1, 8: 2, 9: 2, 10: 2, 11: 2, 12: 2, 13: 2, 14: 2, 15: 3, 16: 3, 17: 3, 18: 3, 19: 3, 20: 3, 21: 3, 22: 3, 23: 3, 24: 4, 25: 4, 26: 4, 27: 4, 28: 4, 29: 4, 30: 4, 31: 4, 32: 4, 33: 5, 34: 5, 35: 5, 36: 5, 37: 5, 38: 5, 39: 5, 40: 5, 41: 5, 42: 6, 43: 6, 44: 6, 45: 6, 46: 6, 47: 6, 48: 6, 49: 6, 50: 7, 51: 7, 52: 7, 53: 7, 54: 7, 55: 7, 56: 7, 57: 7, 58: 7, 59: 8, 60: 8, 61: 8, 62: 8, 63: 8, 64: 8, 65: 8, 66: 8, 67: 8, 68: 9, 69: 9, 70: 9, 71: 9, 72: 9, 73: 9, 74: 9, 75: 10}```

Reply:
Just to be clear, the device uses all 76 different intensity levels, right? It's not that only 11 different levels out of 76 are used in the device?

I think the comment is confusing.

Reply:
This is a guess, but I think they limited the levels in the device UI to 11 since that UI uses two tap buttons (up/down). Tapping up 75 times to go from min to max intensity level isn't a nice UI.

In Home Assistant we don't have that limitation. We often use a percentage range with 0-100 levels which has even smaller difference between each step.

I'm leaning to just using 0-75 as levels. @marcelveldt what do you think? Should we try to mimic the device UI or use whatever makes sense in Home Assistant?

Reply:
Ya, I definitely agree with you about not wanting to tap 75 times.

I'm happy to go either way with it. The code is obvs a lot simpler without the mapping. Just let me know which way you want to go.

Reply:
I would indeed just use a range of 0-75 if the device supports that